packages/aws-client/src/DynamoDb.ts

```diff
@@ -151,37 +163,37 @@ export const scan = improveStackTrace(
  * @param {Object} params
```

Comment:
One last time, change the links on doc to V3

Reply:
6a45893b0fd59ebf7e8b097fab562420c4d90d53